LEAF_Request_Portal/templates/form.tpl

```diff
@@ -114,7 +114,7 @@ function onKeyPressClick(event){
     }
 }
 
-function updateProgress() {
+function updateProgress(focusNext=false) {
     $.ajax({
```

Comment:
error handling? I know this is something that we are trying to at least get console log when errors happen.

Reply:
agree, I will update later with an error callback